rs/nns/governance/init/src/lib.rs

```diff
@@ -289,29 +294,17 @@ impl GovernanceCanisterInitPayloadBuilder {
 
             let neuron_id = NeuronIdProto::from(neuron_id);
 
-            let not_for_profit = record[5]
+            let not_for_profit = record[6]
                 .parse::<bool>()
                 .expect("couldn't read the neuron's not-for-profit flag");
 
-            let memo = if record.len() < 9 {
-                self.rng.next_u64()
-            } else {
-                record[6].parse::<u64>().expect("could not parse memo")
-            };
+            let memo = self.rng.next_u64();
 
-            let maturity_e8s_equivalent = if record.len() < 10 {
-                0
-            } else {
-                record[7].parse::<u64>().expect("could not parse maturity")
-            };
+            let maturity_e8s_equivalent = record[7]
+                .parse::<u64>()
+                .expect("could not parse maturity_e8s_equivalent");
 
-            let kyc_verified = if record.len() < 11 {
-                false
-            } else {
-                record[8]
-                    .parse::<bool>()
-                    .expect("could not parse kyc_verified")
-            };
+            let kyc_verified = false;
```

Comment:
Why remove the ability to specify kyc_verified?

Reply:
Despite this code still being here, the ability to specify kyc_verified has been removed in https://sourcegraph.com/github.com/dfinity/ic/-/commit/b0ea17e4028c309ace18a69257268c05fd4b8ef2